### PR TITLE
(#824) Enable twig for css testing

### LINKED ISSUE #824: Enabler: Enable twigjs in ncids-css-testing for storybook
# Enable twigjs in ncids-css-testing for storybook
> **ESTIMATE** TBD

With importing USWDS native components, we need to be able to setup backstop regressions in order to make sure we do not break USWDS components. The USWDS comes with a number of Storybook stories, however, they use twig for making the HTML. We can implement the same approach and reuse USWDS stories and at the same time remove lots of duplicate copy/paste HTML for our stories. This ticket is to enable twigjs in ncids-css-testing.

### LINKED ISSUE #825: Change usa-skipnav storybook story to use Twig
### Subtasks
- [ ] Create a shared template for usa-skipnav
- [ ] Change the usa-skipnav storybook story to use that shared template

### LINKED ISSUE #827: Change usa-banner storybook stories to use twig
### Subtasks
- [ ] Create a shared template for usa-banner
- [ ] Change the usa-banner storybook story to use that shared template

### LINKED ISSUE #835: Button Group USWDS Pass Through to NCIDS
### USWDS Component Name
[Button Group](https://designsystem.digital.gov/components/button-group/) 

### USWDS Component Packages
* Package usage: `@forward "usa-button-group";`
* Dependencies: `usa-button<\code class=:text-no-wrap"> <\li> </ul>`

### Subtasks
- [x] Make a folder with the component name under uswds-native
- [x] Copy USWDS twig and any JSON
- [x] Make an index.scss for the stories
- [x] Copy and hack up USWDS stories.js file to use our story format
- [x] Create scenarios.js by adding a scenario for each story in stories.js
- [x] Add link to the USWDS documentation on the NCIDS documentation site

### PR DESCRIPTION
**Pull request by olitharp-nci**
Closes #824 
Closes #825 
Closes #827 
Closes #835

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,8 @@
 		"test": "lerna run test",
 		"test:css": "lerna run test:css",
 		"test:a11y": "lerna run test:a11y",
-		"test:css:generate": "lerna run test:css:generate"
+		"test:css:generate": "lerna run test:css:generate",
+		"test:twig": "lerna run twigcs"
```
Comment (bryanpizzillo) [inline package.json:40]: Change `test:twig` to be
`"test:twig": "docker pull ghcr.io/nciocpl/twigcs-docker:v1.0.1  && lerna run twigcs"`

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
See below

**Review by andyvanavery — APPROVED** [review]
This PR correctly enables twig and passes through the Button Group from USWDS to NCIDS per review by Product and Design. This passes review.

File: testing/ncids-css-testing/stories/uswds-native/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Reset editorconfig for uswds files
+root = true
+
+[*.{json,js}]
```
Comment (bryanpizzillo) [inline testing/ncids-css-testing/stories/uswds-native/.editorconfig:4]: This should be json and twig, not js. We own the js and it should follow our standards.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
I think there is just one thing left...

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**